### PR TITLE
[DOC, ENH] allow to use gdb with nvcc. Doc how to use gdb

### PR DESCRIPTION
**Pull request by nouiz**

### CODE REVIEW

File: doc/tutorial/extending_theano_c.txt [hunk 1]
```diff
@@ -983,6 +983,25 @@ macros defined by ``COp``. Finally, the function
...
+
+For this, you must enable this Theano: `cmodule.remove_gxx_opt=True`.
+For the GPU, you must add in this second flag `nvcc.flags=-g` (it slow
+down computation on the GPU, but it is enabled by default on the CPU).
+
+Then you must start Python inside GDB and in it start your Python
+process (e.g. theano-nose):
+
+$gdb python
+(gdb)r bin/theano-nose theano/
```
Comment (lamblin) [inline doc/tutorial/extending_theano_c.txt]: Use `.. code-block:` or `.. code-block: sh` for that section

Comment (nouiz) [inline doc/tutorial/extending_theano_c.txt]: done

**Comment by lamblin** [comment]
OK when travis passes, assuming that you checked that the doc builds and displays correctly.

**Comment by nouiz** [comment]
I checked the output of doc generation. It seem ok. travis have problems, so it will take times to have it pass, but this isn't urgent.

**Pull request merged**